assets/files.wxs

```diff
@@ -3042,6 +3042,15 @@
       <Component Id="cmpCE991A76187F47148F1854BC02E3524E" Guid="{39a0126f-44e9-471a-83d6-139d1636e79a}">
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-windows`
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```